src/blocking/avro.rs

```diff
@@ -1557,4 +1578,40 @@ mod tests {
         };
         assert_eq!(&1i64, counter_value, "counter is 1");
     }
+
+    #[test]
+    fn test_properties() {
```

Comment:
Could you please add a similar test for the async implementation? That will likely also make Codecov happy.